index.bs

```diff
@@ -1650,6 +1764,49 @@ SIGN).</p>
 
 <p>When interpreted as a number, a <a>WebVTT percentage</a> must be in the range 0..100.</p>
 
+<p>A <dfn>WebVTT attributes block</dfn> consists of the following components, in the given order:</p>
+<ol>
+ <li>The string "<code>ATTRIBUTES</code>".</li>
+ <li>
+  The following components, in the given order:
+  <ol>
+   <li>A <a>WebVTT line terminator</a>.</li>
+   <li>Zero or more key/value pairs, parsed in the given order:
+    <ol>
+     <li>A <dfn>WebVTT attribute key</dfn> consisting of <code>[A-Za-z_][0-9A_Za-z_]*</code>:
+      <ul>
+       <li>Any one of the following:
+        <ul>
+         <li>U+0041 LATIN CAPITAL LETTER A through U+005A LATIN CAPITAL LETTER Z</li>
+         <li>U+0061 LATIN CAPITAL SMALL A through U+007A LATIN SMALL LETTER A</li>
```

Comment:
I would reference https://infra.spec.whatwg.org/#ascii-alpha instead of these two bullet points.

Reply:
done